src/Kunstmaan/AdminBundle/Helper/Security/Acl/AclHelper.php

```diff
@@ -204,6 +223,76 @@ private function getPermittedAclIdsSQLForUser(Query $query)
         return $selectQuery;
     }
 
+    /**
+     * @return string
+     */
+    private function getPermittedAclIdsSQLForUserPlatformPostgres(Query $query)
```

Comment:
Why is the whole query duplicated? Can't the solution from [#2811](https://github.com/Kunstmaan/KunstmaanBundlesCMS/pull/2811/files#diff-2ae42119f573a6a61e26009641aa186d97e0f77b951dc02385d67d9daa075854) be used?

Reply:
As a matter of fact I initially did use the solution from #2811 and tested it with one of KBC bank core products. 2811 breaks access control in mysql, due to various issues (amongst which the databaseprefix I think). In order to keep things backwards compatibility with mysql I thiought it prudent to implement a dedicated method for postgres. However as you can see my implementation keeps many of the good stuff of 2811. 
The alternative would have been to place some more (if (dtatabaseplatform==) {}else{} statements in the opriginal code. I do not think that improves code readibility, but perhaps thsi si a matter of tasete

Reply:
I will look into this myself to see what the posibilities are. But my gut feeling is that this is not ideal as this just duplicates the full code. Maybe just inner parts need to be duplicated. But let's see what an ideal solution can be here!